submitting homework2

yanina-prystauka opened the pull request.

Commit:
please accept

Commit:
Add homework files

Commit:
Add timing files

rhancockn commented on DiceCoefSteps.txt:
```diff
@@ -0,0 +1,43 @@
+-modified the proc.sub_03 script
+	- added -tpattern alt+z to line ~90
+	- added -save_all -AddEdge -deoblique on -master_tlrc 3 to line ~107
+	- added -bout to line ~313
```
Ah! This took me awhile to notice, but the problem is that `-bout` changes the index of the contrasts. I ran without `-bout` so my contrast is at index 17. `-bout` adds a lot of other stuff to the stats bucket, so the index has changed (from index 17 to index 63, I think). This means you are computing the overlap for something other than the contrast of interest.  You can rerun the Dice calculations with the stats output you have, but change things like `'stats.sub_03+tlrc[17]'` to `'stats.sub_03+tlrc[T-C_GLT#0_Tstat]'` so that the contrast is correct. Also recall the use of `3dinfo -label` or `3dinfo -label2index` in class today.

Commit:
fixed the 3dmerge command and reran the analyses

yanina-prystauka commented:
Hello, Roeland! I fixed the 3dmerge command and reran the analyses for both 6 mm and 3 mm kernel size, the resulting dice coefficients for both were as expected! And I changed my homework log (DiceCoefSteps.txt), and pushed those changes to make them appear on github.

rhancockn reviewed (commented):
Good!
